crates/core/executor/src/dependencies.rs

```diff
@@ -141,7 +141,7 @@ pub fn emit_memory_dependencies(
             _ => unreachable!(),
         };
 
-        if most_sig_mem_value_byte >> 7 & 0x01 == 1 {
+        if (most_sig_mem_value_byte >> 7) & 0x01 == 1 {
```

Comment:
>  crates/core/executor/src/dependencies.rs:144:12
    |
144 |         if most_sig_mem_value_byte >> 7 & 0x01 == 1 {
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: consider parenthesizing your expression: `(most_sig_mem_value_byte >> 7) & 0x01`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#precedence
    = note: `#[warn(clippy::precedence)]` on by default